styles.css

```diff
@@ -0,0 +1,142 @@
+*,
+*:before,
+*:after {
+  -webkit-box-sizing: border-box;
+     -moz-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+.container {
+  margin: 0 auto;
+  width: 100%;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.content {
+  margin: 25px auto;
+  width: 80%;
+}
+
+.logo {
+  display: inline-block;
+  width: 20%;
+}
+
+.logo img, .logo svg {
+}
+
+.search {
+  display: inline-block;
+  width: 60%;
+  margin: 15px 0;
+}
+
+.search input {
+  width: 80%;
+  padding: 7px;
+  border: 1.5px solid #EFEFEF;
+}
+hr {
+  border: 1px solid #EFEFEF;
+  margin: 25px 0;
+}
+
+.search img {
+  position: absolute;
+  /* padding: 5px; */
+}
+
+.social {
+  display: inline-block;
+  width: 20%;
+}
+
+.social button {
+  float: right;
+}
+
+header {
+  width: 100%;
+  box-shadow: 0 8px 15px 0 rgba(0,0,0,0.1);
+  transition: 0.3s;
+}
+
+
+main{
+  /* float: left; */
+  display: inline-block;
+  width: 70%;
+  /* background-color: aquamarine; */
```

Comment:
It is better to delete commented code :smiley:

Reply:
Done!